src/main/java/baseEntities/BaseTest.java

```diff
@@ -0,0 +1,31 @@
+package baseEntities;
+
+import drivers.BrowserType;
+import drivers.DriverFactory;
+import drivers.DriverManager;
+import org.openqa.selenium.WebDriver;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class BaseTest {
+
+    public static WebDriver getDriver() {
+        return DriverManager.getDriver();
+    }
+
+    @BeforeMethod
+    public void setUp(){
+        try {
+            DriverManager.setWebDriver(DriverFactory.getDriver(BrowserType.OPERA));
+        } catch (Exception e) {
```

Comment:
Why do we need try {} catch {} here? DriverManager.setWebDriver doesn't throw any exceptions

Reply:
fixed